contents/docs/migrate/migrate-between-posthog-instances.mdx

```diff
@@ -42,3 +42,10 @@ Follow the instructions on the [PostHog migrate metadata repo](https://github.co
     ```
 
 For more information on the options see the [repo's readme](https://github.com/PostHog/posthog-migrate-meta)
+
+## Step-by-step migration video tutorial
+
+<iframe
+  src="https://www.youtube-nocookie.com/embed/INFtGMraiBg"
+  className="rounded shadow-xl"
+/>
```

Comment:
Nit: I would put the video at the top, and not at the bottom since people shouldn't have to read the entire doc to find out there's a video version too. 

I cant add a code suggestion, but I would insert it just below the callout:
```
> If you're attempting this migration, feel free to ask questions and provide feedback via the [PostHog Communty Slack workspace](/slack) or [a GitHub issue](https://github.com/PostHog/posthog.com/issues). You can also [raise a support ticket in the app](https://app.posthog.com/home#supportModal). 

<iframe
  src="https://www.youtube-nocookie.com/embed/INFtGMraiBg"
  className="rounded shadow-xl"
/>
```

Reply:
good points, updated